src/components/App.js

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
-
+import React, { Component } from 'react';
 import Router from './Router';
 
-const App = () => <Router />;
+import { catchErrorsWithScope } from 'utils/sentryDSN';
```

Comment:
Let's just name the utility `sentry`.

```suggestion
import { catchErrorsWithScope } from 'utils/sentry';
```

Reply:
done!